CodeSniffer/Standards/PEAR/Sniffs/Commenting/FileCommentSniff.php

```diff
@@ -347,6 +347,12 @@ protected function processPackage(PHP_CodeSniffer_File $phpcsFile, array $tags)
             $newContent = preg_replace('/[^A-Za-z_]/', '', $newContent);
             $nameBits   = explode('_', $newContent);
             $firstBit   = array_shift($nameBits);
+            
+            if(!isset($firstBit{0})){
```

Comment:
But what `$firstBit{0}` technically does? I thought that it only can be used on objects.